templates/foreman-tasks.yaml.erb

```diff
@@ -0,0 +1,30 @@
+:foreman-tasks:
+#
+# Logging configuration can be changed by uncommenting the loggers
+# section and the logger configuration desired.
+#
+  :loggers:
+    :dynflow:
+      :enabled: <%= scope.lookupvar("::foreman::plugin::tasks::logger_dynflow") %>
```

Comment:
Since the variables are in scope you can use ```<%= @logger_dynflow %>```. The same goes for all other config values used here.